hat/assets/js/apps/Iaso/domains/forms/fields/hooks/useGetQueryBuildersFields.ts

```diff
@@ -27,7 +27,7 @@ export const useGetQueryBuildersFields = (
         ) {
             fields[field.fieldKey] = {
                 ...currentField.queryBuilder,
-                label: formatLabel(field),
+                label: formatLabel(field) + ' [' + field.name + ']',
```

Comment:
```suggestion
                label: `${formatLabel(field)} [${field.name}]`,
```
more clear and more readable